src/Sentry.Unity.Android/SentryNativeAndroid.cs

```diff
@@ -39,6 +39,16 @@ public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentry
 
         JniExecutor ??= new JniExecutor();
 
+        if (SentryJava.IsEnabled(JniExecutor))
+        {
+            options.DiagnosticLogger?.LogDebug("The native SDK is already initialized");
```

Comment:
Keep terminology consistent (see log msg below)

```suggestion
            options.DiagnosticLogger?.LogDebug("The Android Native SDK is already initialized");
```